source/release-notes/backend.txt

```diff
@@ -8,6 +8,18 @@ Atlas App Services Changelog
    :depth: 1
    :class: twocols
 
+.. _backend-2023:
+
+2023 Releases
+-------------
+
+.. _backend_20230111:
+
+11 January 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Configure user creation function via the App Services CLI and Admin API.
```

Comment:
TODO: add ref on merge of changes for https://jira.mongodb.org/browse/DOCSP-26904

Reply:
It looks like [that related PR](https://github.com/mongodb/docs-app-services/pull/301) got merged - do you wanna add a link?

Reply:
```suggestion
- Configure :ref:`user creation function <user-creation-function>` via the App Services CLI and Admin API.
```